app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass

```

Comment:
Consider providing a descriptive error message for the VaccineError exception. According to the checklist, using descriptive error messages is a good practice.